packages/flutter_reactive_ble/lib/src/device_scanner.dart

```diff
@@ -1,9 +1,6 @@
 import 'dart:async';
 
 import 'package:flutter_reactive_ble/flutter_reactive_ble.dart';
-import 'package:flutter_reactive_ble/src/rx_ext/repeater.dart';
```

Comment:
I am always a bit more fan of specific imports than the full package import. However it is more a preference not a blocker to me